ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/site/js/tabs.js

```diff
@@ -269,34 +266,50 @@
          * @private
          */
         #refreshActive() {
-            var tabpanels = this._elements["tabpanel"];
-            var tabs = this._elements["tab"];
-
+            var tabpanels = this.#getCachedTabPanels();
```

Comment:
newly added tab will not be available.

Reply:
Handled that scenario sakshi, we recache the elements on instance addition.
In handleChildAddition method, once the markup of newly added element is handled , then this is called
this.#cacheElements(this._elements.self);